graphite_api/node.py

```diff
@@ -24,8 +24,11 @@ def __init__(self, path, reader):
         self.intervals = reader.get_intervals()
         self.is_leaf = True
 
-    def fetch(self, startTime, endTime):
-        return self.reader.fetch(startTime, endTime)
+    def fetch(self, startTime, endTime, maxDataPoints=None):
+        if hasattr(self.reader, '__aggregating__'):
+            return self.reader.fetch(startTime, endTime, maxDataPoints)
+        else:
+            return self.reader.fetch(startTime, endTime)
```

Comment:
Can you use [`inspect.signature`](https://docs.python.org/3/library/inspect.html#inspect.signature) (or [`getargspec`](https://docs.python.org/2/library/inspect.html#inspect.getargspec) on Python 2) to "guess" the ability of `.fetch` to accept maxDataPoints instead of having to rely on a magic attribute?

Or do a `try… except TypeError` but it would be less elegant…